test/utils/tokenize.ts

```diff
@@ -361,6 +361,25 @@ export namespace Token {
             export const Binary = (text: string) => createToken(text, 'constant.numeric.binary.cs');
             export const Decimal = (text: string) => createToken(text, 'constant.numeric.decimal.cs');
             export const Hexadecimal = (text: string) => createToken(text, 'constant.numeric.hex.cs');
+            export const Invalid = (text: string) => createToken(text, 'invalid.illegal.constant.numeric.cs')
```

Comment:
I'd have put this under some specific namespace for invalid tokens, but there was none. 

The only other similar one was `IllegalNewLine - invalid.illegal.newline.cs` but it is directly under the `Token` namespace. I could potentially refactor this to have them both grouped under the same `Token.Invalid` or `Token.Illegal` namespace.